src/js/Controllers/UIController.js

```diff
@@ -529,17 +540,36 @@ class UIController {
         this.listener.send(RpcFactory.OnButtonPressNotification(appID, button))
     }
     failInteractions() {
+        const state = store.getState()
+        var activeApp = state.activeApp
+        var app = state.ui[activeApp]
+        var interactionId = app.interactionId
         for (var msgID in this.timers) {
             clearTimeout(this.timers[msgID])
             delete this.timers[msgID]
             RemoveImageValidationResult(msgID)
+            if (msgID === interactionId && app.interactionLayout === "KEYBOARD") {
```

Comment:
The `msgID === interactionId` check fails here since `msgID` and `interactionId` are not the same type.(`typeof msgID` is string and `typeof interactionId` is number)

Reply:
Wow nice catch!

Updated in 
d556906